src/suricata.c

```diff
@@ -1467,6 +1467,16 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
                 AppLayerParserSetup();
                 RegisterModbusParsers();
                 exit(AppLayerParserFromFile(ALPROTO_MODBUS, optarg));
+            } else if(strcmp((long_opts[option_index]).name, "afl-enip-request") == 0) {
```

Comment:
these options don't work as the options are not registered in the long_opts struct above. Please test things before submitting a PR.